trailblazer/mip/config.py

```diff
@@ -16,7 +16,7 @@ class SampleSchema(Schema):
     sample_id = fields.Str(required=True)
     analysis_type = fields.Str(
         required=True,
-        validate=validate.OneOf(choices=['wes', 'wgs', 'tga'])
+        validate=validate.OneOf(choices=['wes', 'wgs', 'tga', 'wts'])
```

Comment:
This will allow MIP to run wts samples, shouldn't we have a separate schema for mip-rna?

Reply:
True that